<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="C02FXF4AMD6M" failures="2" tests="9" name="Default test" time="69.721" errors="0" timestamp="2024-07-08T16:29:31 CDT">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.133"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.577"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.139"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.152"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.574">
    <failure type="java.lang.AssertionError" message="Maximum limits are reached and user can no longer log in expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Maximum limits are reached and user can no longer log in expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertFalse(Assert.java:67)
at PageModels.LoginPage.clickNextAndCheck(LoginPage.java:75)
at stepdefinitions.LoginUISteps.user_enters_examples_credentials(LoginUISteps.java:111)
at ✽.user enters "likithal39@gmail.com" and "Sun$Daisy132"(file:///Users/likitha.lokesh/java-selenium-cucumber/src/test/resources/features/LoginUI.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.821"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="14.996"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="30.126"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.789">
    <failure type="java.lang.AssertionError" message="Maximum limits are reached and user can no longer log in expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Maximum limits are reached and user can no longer log in expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertFalse(Assert.java:67)
at PageModels.LoginPage.clickNextAndCheck(LoginPage.java:75)
at stepdefinitions.LoginUISteps.the_user_enter_sheet_and(LoginUISteps.java:98)
at ✽.The user enter sheet "login" and 1(file:///Users/likitha.lokesh/java-selenium-cucumber/src/test/resources/features/LoginUI.feature:41)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
